File: app/src/main/java/com/code_design_camp/client/friday/HeadDisplayClient/Util/Connectivity.java
package com.code_design_camp.client.friday.HeadDisplayClient.Util;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

public class Connectivity {
    public static boolean isConnected(Context c) {
        ConnectivityManager cm = (ConnectivityManager) c.getSystemService(c.CONNECTIVITY_SERVICE);
        NetworkInfo networkInfo = cm.getActiveNetworkInfo();
        if (networkInfo != null && networkInfo.isConnected()) {
            return true;
        }
        return false;
    }
}

File: appModules/core/src/main/java/com/friday/ar/core/util/net/Connectivity.kt
package com.friday.ar.core.util.net

import android.content.Context
import android.net.ConnectivityManager

object Connectivity {
    fun isConnected(c: Context): Boolean {
        val cm = c.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        val networkInfo = cm.activeNetworkInfo
        return networkInfo != null && networkInfo.isConnected
    }
}
